tsrelay/handler/get_serve.go

```diff
@@ -53,7 +62,7 @@ func (h *handler) getServeHandler(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(s)
 }
 
-func (h *handler) getServe(ctx context.Context, body io.Reader) (*serveStatus, error) {
+func (h *handler) getServe(ctx context.Context, body io.Reader, withPeers bool) (*serveStatus, error) {
```

Comment:
If we're using this for both the status and serve status, should we rename this to `getStatus` and make an option to include the serve config?

Reply:
@tylersmalley yep I had that in mind, but I'm actually still not sure if we should separate them into two separate endpoints. So I'll keep this for now with a TODO because I just want us to unblock the error handling first.